Dockerfile

```diff
@@ -1,4 +1,37 @@
-FROM python:3.9 AS downloader
+################################
+# python-base
+# Set up shared environment variables
+################################
+FROM python:3.9 as python-base
+
+    # Poetry
+    # https://python-poetry.org/docs/configuration/#using-environment-variables
+ENV POETRY_VERSION=1.7.0 \
```

Comment:
is there a reason to have a multi-line with `\`s here as opposed to many `ENV` statements? it looks not very ~~pythonic~~ dockerrific

Reply:
This was just usually how I saw it done in most examples. Granted I don't know Docker too well-- does each `ENV` statement create one layer? If so, I could see how it would be more straightforward to the final image if we have one `ENV` statement that does all of our poetry configuration. I figured that is why we did this for multiline `RUN` statements that are related commands.

Reply:
sounds good then